sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,20 @@
+-- Ragged John
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = 9563;
+
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 9563);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(9563, 0, 0, 1, 10, 0, 100, 0, 0, 15, 0, 0, 11, 16472, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Ragged John - Within 0-15 Range Out of Combat LoS - Cast \'Wicked Milking\''),
+(9563, 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 15, 4866, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Ragged John - Within 0-15 Range Out of Combat LoS - Quest Credit \'Mother\'s Milk\'');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceGroup` = 1 AND `SourceEntry` = 9563 AND `SourceId` = 0;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(22, 1, 9563, 0, 0, 1, 0, 16468, 1, 0, 0, 0, 0, '', 'SAI event 0 for Ragged John will execute if target has aura Mother\'s Milk.'),
+(22, 1, 9563, 0, 0, 32, 0, 16, 0, 0, 0, 0, 0, '', 'SAI event 0 for Ragged John will execute if target is player.');
+
+-- Missing events?
+
+-- per wowhead - player dialog w/NPC "Milk me, John." "With pleasure... just let me savor this moment for a minute longer." "Do it... do it now."
+-- verifiable w/ broadcast_text id 5833, 5834, 5835
+-- no video available that I can find, unfortunately, so I have no idea how this relates at all to the actual event action above, or whether that may be handled by the spell (I'll check on this before PR is opened.)
```

Comment:
```sql
SELECT * FROM `broadcast_text` WHERE `ID` BETWEEN 5833 AND 5835;
```
```
    ID  Language  MaleText                                                             FemaleText           
------  --------  -------------------------------------------------------------------  ---------------------
  5833         0  Milk me, John.                                                       Milk me, John.       
  5834         0  With pleasure... just let me savor this moment for a minute longer.                       
  5835         0  Do it... Do it now.                                                  Do it... Do it now.  
```

Quite possible he should cast the spell only after player select the gossip.

Reply:
Could be gossip, in which case this whole thing becomes a lot easier. I figured based on that Wowhead comment it was scripted /say text - just based on the word "dialogue" - but I was hoping someone would have a video somewhere - without other sources, I usually default to doing a direct conversion from the C++ script.

I'm fine with moving it to gossip if devs are also comfortable with that - maybe we can get a check in from them on that :)

Reply:
One text already present in npc_text, so..
```sql
SELECT * FROM `npc_text` WHERE `ID`=3496;
```
```text
    ID  text0_0                                                              
------  ---------------------------------------------------------------------
  3496  With pleasure... just let me savor this moment for a minute longer.  
```

Reply:
good catch, that's enough for me to be going on with! I'll make necessary changes sometime today or tomorrow.